Answers/hw03_stubs_do_not_edit.py

```diff
@@ -0,0 +1,85 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Megan Lucas
+# Username:Megan_Lucas
+#
+# Assignment:HW03_stub
+# Purpose: To build a bouquet of flowers with a bow
+# Google Doc Link:https://docs.google.com/document/d/1ZtouqxWRp3KCr7y-horbUReL1Hd0j0SWmor7oxk6oxs/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor("lightblue")
+m = turtle.Turtle()
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor("lightblue")
+m = turtle.Turtle()
+m.pendown
+### first stem
+m.speed(0)
+m.color("green")
+m.left(90)
+m.forward(100)
+m.right(180)
+m.forward(50)
+### second stem
+m.right(-45)
+m.forward(-20)
+m.right(180)
+m.forward(50)
+### third stem
+m.right(180)
+m.forward(70)
+m.left(90)
+m.forward(70)
+
+
+def draw_petal(petal1):
+    m.color("red")
+    # finding position of the stem
+    # m.goto(60, 100)
+    m.circle(10, 150)
+    m.left(120)
+
+
+def draw_flower(petal1):
+    for i in range(petal1):
+       draw_petal(petal1)
+       m.right(360/petal1)
+
+
+
+def main():
+#middle flower drawing
+    m.penup()
+    m.goto(0, 100)
+    m.pendown()
+    draw_flower(20)
```

Comment:
Here is where you need to pass in the turtle!

```
draw_flower(20, m)
```

Your function definition and your function call are a pair. The order of parameters should match:

```
def draw_flower(sides, petal1):
  """
  Draws a flower
  :param: sides: the number of sides of a polygon
  :param: petal1: a turtle object to draw the flower with
  """
```